extras/webClient/lib/view/information_page.dart

```diff
@@ -89,20 +158,47 @@ class Information extends StatelessWidget {
               ),
             ),
           );
-          return ListView(
-            children: <Widget>[
-              DataTable(
-                headingRowHeight: 0,
-                columns: const <DataColumn>[
-                  DataColumn(
-                    label: Text('Key'),
-                  ),
-                  DataColumn(
-                    label: Text('Value'),
-                  ),
-                ],
-                rows: informationRows,
-              )
+          return Column(
+            children: [
+              Expanded(
+                child: ListView(
+                  children: <Widget>[
+                    DataTable(
+                      headingRowHeight: 0,
+                      columns: const <DataColumn>[
+                        DataColumn(
+                          label: Text('Key'),
+                        ),
+                        DataColumn(
+                          label: Text('Value'),
+                        ),
+                      ],
+                      rows: informationRows,
+                    ),
+                  ],
+                ),
+              ),
+              // This outer ternary is required to prevent the screen flashing an error
+              // for a split second while appData.information loads to do the check.
+              (appData.information['Version'] != null)
+                  ? OutlinedButton(
+                      onPressed: versionCheck(appData.information['Version'])
+                          ? () {
+                              unawaited(
+                                startWebFilePicker(
```

Comment:
Can we limit by file type and size?

Reply:
I have added these in. Currently it is limited to file types of .rtf, .txt, and .csv that are less than 10KB in size, though this can be changed if needed.